Description: Update site-specific configuration path in man page
 In Debian, there is exactly one site-specific configuration directory, and it
 is known in advance.  So we can safely remove the list of possible paths from
 the man page in order to avoid confusion.
Author: Noah Meyerhans <noahm@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=426031
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: spamassassin/spamassassin.raw
===================================================================
--- spamassassin.orig/spamassassin.raw
+++ spamassassin/spamassassin.raw
@@ -670,27 +670,7 @@ in:
 =back
 
 Site-specific configuration data is used to override any values which had
-already been set.  This is loaded from the first existing directory in:
-
-=over 4
-
-=item @@LOCAL_RULES_DIR@@
-
-=item @@PREFIX@@/etc/mail/spamassassin
-
-=item @@PREFIX@@/etc/spamassassin
-
-=item /usr/local/etc/spamassassin
-
-=item /usr/pkg/etc/spamassassin
-
-=item /usr/etc/spamassassin
-
-=item /etc/mail/spamassassin
-
-=item /etc/spamassassin
-
-=back
+already been set.  This is loaded from /etc/spamassassin.
 
 From those directories, SpamAssassin will first read files ending in
 ".pre" in lexical order and then it will read files ending in ".cf" in
